xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md

```diff
@@ -0,0 +1,88 @@
+
+We decided to follow suit to close a window of opportunity for hard to find bugs. Therefore, the version 2.24 of Xtext and Xtend which is due along with Eclipse 2020-12 will be the first release that depends on Java 11. Please feel encouraged to join the discussion in [https://github.com/eclipse/xtext/issues/1804](https://github.com/eclipse/xtext/issues/1804) if you have any concerns, general comments or other suggestions.
+
+## UI Testing
+
+The AbstractCodeMiningTest and the AbstractMultiQuickfixTest base classes have been added to the org.eclipse.xtext.ui.testing package to provide a convenient way to test the code mining and multi-quickfix capabilities. The Xtext example projects and the Xtend code base have been extended by concrete test cases to demonstrate the usage of these framework classes.
+
+## Junit 5
+
+`InjectionExtension` now supports the Junit 5 `@Nested` tests.
+
+## Core API
+
+A protected method to get at the lazy proxy information of a LazyLinkingResource was added to enable subclasses to work with these.
+
+## Eclipse Integration
+
+### Quickfix improvements
+
+Convert a terminal fragment to a terminal rule. It resolves the following errors:
+
+* A rule call in the parser rule points to a terminal fragment.
+* A hidden token in the grammar definition or a parser rule points to a terminal fragment.
+
+Remove illegal hidden token definition. It resolves the following errors:
+
+* The hidden-token points to a parser rule, terminal fragment, or an enum rule.
+
+Create a new rule definition. It resolves the following errors:
+
+* An undefined parser rule is cross-referenced.
+* An undefined enum, terminal, or a terminal fragment is cross-referenced.
+
+([xtext-eclipse#1448](https://github.com/eclipse/xtext-eclipse/issues/1448))
+
+### Eclipse API
+
+The class org.eclipse.xtext.ui.editor.reconciler.TemplatePositionUpdater is now public to be usable for clients.
+
+## Upgrades
+
+* Eclipse platform dependencies where updated (e.g org.eclipse.core.runtime:3.17.100 to 3.18.0).
+
+## Credits
+
+The Xtext project is thankful for the dedication of each committer and contributor. This release has been made possible by the following persons (in order of the number of contributed commits to this release to [all repositories](https://github.com/eclipse/xtext#repositories) except [xtext-xtend](https://github.com/eclipse/xtext-xtend)):
+
+- Christian Dietrich (itemis)
+- Nico Prediger (itemis)
+- Sebastian Zarnekow (independent)
+- Karsten Thoms (Karakun)
+- Tamas Miklossy (itemis)
+- Arne Deutsch (itemis)
+- Jan Koehnlein (TypeFox)
+- Neeraj Bhusare (Zafin)
```

Comment:
Not sure if this is conventional but we might want to do a small shoutout that Neeraj has become an official commiter

Reply:
We did not announce new committers in release notes in the past, but more in blog posts. And just formally, the nomination process did not conclude yet. Of course, his addition is highly appreciated.